Bio/AlignIO/MafIO.py

```diff
@@ -166,6 +166,10 @@ def MafIterator(handle, seq_count=None):
                     "size": int(line_split[3]),
                     "strand": strand,
                     "srcSize": int(line_split[5]),
+                    "leftStatus": None,
+                    "leftCount": None,
+                    "rightStatus": None,
+                    "rightCount": None,
```

Comment:
Do you think it is useful to always have these in the dictionary, even if absent from the file being parsed? I would have added them as required.

Reply:
Indeed this was one of my concern. Is it better to have it with default value or to have it only when provided.

Reply:
I don't know your standards. If you think it is better not to have it. I can change it.

Reply:
If most MAF files lack the "i" lines, then personally I would leave them out of the dictionary by default.

Reply:
OK I change it.